fog/ledger/server/src/db_fetcher.rs

```diff
@@ -30,18 +31,30 @@ use std::{
 const TELEMETRY_BLOCK_INDEX_KEY: Key = telemetry_static_key!("block-index");
 
 /// An object for managing background data fetches from the ledger database.
-pub struct DbFetcher {
+pub struct DbFetcher<
+    DB: Ledger + 'static,
+    E: LedgerEnclaveProxy + Clone + Send + Sync + 'static,
+    SS: ShardingStrategy + Send + Sync + 'static,
+> {
```

Comment:
Do we want to maintain compatibility with single-node ledger server, for the sake of moving over to router/store? Is there a way to initialize this for that use case?

Reply:
As per other discussions, the single-node configuration will no longer be supported.